### ISSUE TITLE
standard:kdoc - A KDoc is not allowed inside 'value_argument_list'

### ISSUE THREAD

**Issue by vanniktech** [pinterest/ktlint#2656]
The following enum entry gets flagged with 1.2.1:

```kotlin
      /** Planks. */
      Exercise(
        ...
      )
```

Why isn't kdoc allowed on an individual enum entry?

**Comment by paul-dingemans** [comment]
Please share a more complete code example and your `.editorconfig` that reproduces finding above.

Examples below are accepted by Ktlint without warnings:
```
enum class FooBar {
    /** Some foo */
    FOO,

    /** some bar */
    BAR
}

enum class FooBar2(s: String) {
    /** Some foo */
    FOO("aa"),

    /** Some bar */
    BAR("bb")
}
```
with `.editorconfig`:
```
root = true

[*.{kt,kts}]
ktlint_code_style = ktlint_official
ktlint_standard = enabled
ktlint_experimental = enabled
```

**Comment by paul-dingemans** [comment]
Closed due to lack of response.
